docs/components/receive.md

```diff
@@ -15,6 +15,14 @@ Thanos Receive supports multi-tenancy by using labels. See [Multitenancy documen
 For more information please check out [initial design proposal](../proposals/201812_thanos-remote-receive.md).
 For further information on tuning Prometheus Remote Write [see remote write tuning document](https://prometheus.io/docs/practices/remote_write/).
 
+As of version [v0.22.0](https://github.com/thanos-io/thanos/blob/main/CHANGELOG.md#added), the Thanos receiver implements the [receiver split proposal](https://github.com/thanos-io/thanos/blob/main/docs/proposals/202012_receive_split.md), which allows data ingestion as well as request routing to be independently enabled/disabled.
+
```

Comment:
> This would enable users to prepare a topology of receivers, containing trees of receiver with depth **N**.

Why would users want to do that? The documentation is the place that users will look for guidance on this topic.

One thing that comes to mind is that this lets the users achieve hard tenancy of receivers - no two customers will share receiver ingest instances.

Reply:
The receiver split [proposal](https://github.com/thanos-io/thanos/blob/main/docs/proposals/202012_receive_split.md) does a good job of explaining why.

With the changes coming in in `v0.22.0`, only routing-receivers need to watch for configuration changes, and ingesting-receivers do not need to. So when the configuration is updated, the routing-receivers will almost instantaneously re-configure without a lengthy WAL flush (& downtime) like we have in the current setup (routing & ingesting receivers).

Reply:
Shall I add a line at the end -

> If you are curious about why you should setup your receiver topology, please refer the proposal for more details.

Reply:
> With the changes coming in in v0.22.0, only routing-receivers need to watch for configuration changes, and ingesting-receivers do not need to. So when the configuration is updated, the routing-receivers will almost instantaneously re-configure without a lengthy WAL flush (& downtime) like we have in the current setup (routing & ingesting receivers).

Also, I think we can add your description in the doc. The above statement sounds descriptive about the use case. cc @squat WDYT?

Reply:
I think we need to use full sentences, otherwise sgtm (:

Reply:
> I think we need to use full sentences, otherwise sgtm (:

I didn't get which sentence we should complete, can you please explain 😅